### ISSUE TITLE
New address field format - single line

### ISSUE THREAD

**Issue by KoalaMango** [CRUKorg/pl-sprint-prototype#16]
## What
> Give a brief description of the pattern you want to propose.

New address field format - single line

## Why
> Explain why you think this should be added to the CRUK Pattern library. 

* What evidence do you have that it's needed by multiple services across CRUK?
Performs exactly the same as old one, but takes less time for user to review ( different label format) and makes forms look shorter

* What evidence do you have that it meets the needs of the users of those services?
User testing results

* Have you checked that it doesn't already exist in the CRUK Pattern library?
No

## Anything else

* Include links to any examples, research or code to support your proposal, if available.
First iteration is live here: https://activity.cancerresearchuk.org/signup/cycle-300?channel=cycle____
Second iteration hiding country is on it's way. I can share screenshots.

* Include your pull request link
N/a

* Your email address or Github username
karolina.czarnecka@cancer.org.uk

**Comment by DylanUX** [comment]
When Pattern Library (PL) is live submissions should probably only be reviewed if the answer to 'have you checked the PL?' is yes.

Should we not allow the form to submit if this is 'no' and advise user to go check? Can we do this without them losing the info they have already entered?

**Comment by DylanUX** [comment]
@Karolina (why did tagging not work here?) 

RE: Your response to 'What evidence do you have that it meets the needs of the users of those services?' 
Could you please provide more details. Ideally I would like to see some context and metrics.

Something like:
_Usability tested on 23/05/2018 with 6 Women aged between 25 and 60
5 out of 6 users completed the task of entering their address very easily.
The 6th user had a little difficulty but managed to complete the task in a reasonable time.

We asked 6 users to review the old address fields [insert link], and 6 to review the new pattern. 
The new pattern took, on average 8 seconds less to accurately review._

This type of detail then informs others who may wish to challenge the pattern of the metrics used and what they need to improve upon. They could also challenge for metrics and / or method.

It will (assumption) also provide people with more confidence in the pattern and the PL as a whole.

This month the UX team is trialing entering details of all tests and experiments into a one pager. If this is adopted across Tech and the organisation a link to one or more of these docs may also be required here.

Research one pager examples: https://cancerresearchuko365.sharepoint.com/:f:/s/Tech-UserExperienceUX/Ei7spS7Ji4hKpZuFQO9eVYgBu3Ds6wNETkaodOw7tFOmXw?e=8ywVin

Hope this makes sense and thank you so much for being so engaged.

Any thoughts much appreciated

**Comment by karolinacza** [comment]
@DylanUX Makes sense. I haven't been around when the User Testing was done. I only got one liner update from Viki and Rachal handover. Fist iteration of the story was already done and wasn't linked to any UX documents. I will need to get Joe Cooling to provide more info.
